### PR TITLE
Feature/google maps start

### PR DESCRIPTION
**Pull request by bhohnco**
## Type of Change
- [ ] JSX
- [ ] HTML
- [X] JavaScript
- [X] CSS
- [X] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
Create a MapView component that is hooked up to Google Maps API 
The Map is showing on the main page view
## Please give a short description of the changes that were made below

- Fix:

- Feature: The Google Map view is showing on the main page, and linked to the Google Maps API. We will need to switch this out once we hook up the BE server 

## How has this been tested?
- [ X ] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

**Review by carolineshowscode — APPROVED** [review]
Nice job figuring this out!

File: walk-safe/package.json [hunk 1]
```diff
@@ -4,14 +4,17 @@
   "private": true,
   "dependencies": {
     "@apollo/client": "^3.3.21",
+    "@atomap/use-places-autocomplete": "^1.0.4",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "apollo-server": "^2.25.2",
+    "google-maps-react": "^2.0.6",
```
Comment (pcmueller) [inline walk-safe/package.json:12]: Thx for reinstalling these packages - hopefully that'll get us all on the same page.

File: walk-safe/src/Components/MapView/MapView.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const mapStyles = {
+  width: '40%',
+  height: '40%'
+};
+
+export class MapView extends Component {
+  render() {
+    return (
+      <div className='map-container'>
+        <Map
```
Comment (pcmueller) [inline walk-safe/src/Components/MapView/MapView.js:13]: Psyched about this map view - looking forward to hearing more about how it's structured!

**Review by pcmueller — APPROVED** [review]
Looks great Bryan - thanks for getting our Map View component up and running.

**Review by joemecha — APPROVED** [review]
Great! Thanks for sharing the new changes.

**Pull request merged**